tests/helpers/stubs.py

```diff
@@ -222,19 +222,37 @@ def pos_perc(self):
         return self._pos_perc
 
 
+class FakeWebTabHistory(browsertab.AbstractHistory):
+
```

Comment:
My bad, but this needs a docstring (something like `"""Fake for Web{Kit,Engine}History."""` or so)